src/pybind/ceph_argparse.py

```diff
@@ -502,13 +502,13 @@ def valid(self, s, partial=False):
         try:
             poolid = int(poolid_s)
         except ValueError:
-            raise ArgumentFormat('pool {0} not integer'.format(poolid))
+            raise ArgumentFormat('pool {0} not integer'.format(poolid_s))
```

Comment:
f-strings are a better alternative here instead of format() method but looking at the entire `pybind/ceph_argparse.py` there are 70 usages of format() so merely changing these two occurrences might look odd, it could be a good idea to refactor the code but I don't think this is of high prio(?), just a heads up.

Reply:
Yeah, I agree. I didn't change it for the same reason you mentioned. let me do this change in a separate commit/separate PR. @vshankar ?